sdk/deviceupdate/Azure.Iot.DeviceUpdate/src/Generated/Models/PageableListOfDeviceClasses.Serialization.cs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+// <auto-generated/>
+
+#nullable disable
+
+using System.Collections.Generic;
+using System.Text.Json;
+using Azure.Core;
+
+namespace Azure.Iot.DeviceUpdate.Models
+{
+    public partial class PageableListOfDeviceClasses
```

Comment:
These classes shouldn't be generated but it's a code-gen bug. https://github.com/Azure/autorest.csharp/pull/1066

Reply:
Got it, what should I do about that?

Reply:
The fix has landed in master, rebase and re-generate.